src/components/PaginationBar.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./PaginationBar.css";
+import LeftArrorw from "../assets/left-arrow.svg";
+import RightArrow from "../assets/right-arrow.svg";
+
+const PaginationBar = ({ totalPageNum, activePageNum, onPageChange }) => {
+  const maxVisiblePages = 5; // 한 번에 보여줄 페이지의 최대 개수
```

Comment:
상수는 컴포넌트 밖으로 빼는 걸 추천드려요!
```suggestion
const MAX_VISIBLE_PAGES = 5; // 한 번에 보여줄 페이지의 최대 개수

const PaginationBar = ({ totalPageNum, activePageNum, onPageChange }) => {
```